tests/pyqtsignal.py

```diff
@@ -5,11 +5,25 @@
 class Class(QtCore.QObject):
     signal = QtCore.pyqtSignal([str])  # type: typing.ClassVar[QtCore.pyqtSignal]
 
+    def __init__(self) -> None:
+        super().__init__()
```

Comment:
Does defining this have an affect vs. letting the `QObject.__init__()` handle it?

Reply:
I can not reproduce right now, but I remember going crazy at first when that line was not present. Mypy would refuse to consider instance.signal as a typed variable and would output "Any" when using reveal_type(). The only way to fix that problem was to add this explicit initialization.

The problem is that assigning incorrectly Any to a type is not detected by the current test. I just checked, removing the __init__ line does not trigger the behavior I saw, but I remaing wary of it. So, I prefer to keep it.